features/constructed-stylesheets.yml

```diff
@@ -3,13 +3,12 @@ description: The `CSSStyleSheet` constructor creates a new stylesheet which can
 spec:
   - https://drafts.csswg.org/cssom-1/#dom-cssstylesheet-cssstylesheet
   - https://drafts.csswg.org/cssom-1/#dom-documentorshadowroot-adoptedstylesheets
+status:
+  compute_from: api.CSSStyleSheet.CSSStyleSheet
 compat_features:
   - api.CSSStyleSheet.CSSStyleSheet
   - api.CSSStyleSheet.replace
   - api.CSSStyleSheet.replaceSync
   - api.Document.adoptedStyleSheets
   - api.ShadowRoot.adoptedStyleSheets
-  # Setting the base URL is probably not important for typical uses of
```

Comment:
Same here, and elsewhere. It would be nice if we had a way to keep these editorial notes in there somewhere, maybe in a more structured form.
It would be nice to see them in the dist files in fact.What do you think?

Reply:
> It would be nice if we had a way to keep these editorial notes in there somewhere, maybe in a more structured form.

I agree that it would be nice, though I think `compute_from` is going to handle the most common case, by marking the boundary after which things are later additions. So I'm doubtful that every comment saying "this is a later addition" will make sense to maintain.


Additionally, a lot of `compat_features` lists will eventually live as tags in BCD. So we'd need a separate way to list compat key commentary.

Reply:
Agreed that just having a comment that says "later addition" isn't useful. I'm more concerned about the various links and pieces of info that we spent time digging up when researching a feature.